DopplerJobsServer/Properties/launchSettings.json

```diff
@@ -17,7 +17,7 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
-    "Server": {
+    "Doppler.Job.Server": {
```

Comment:
Is it fine?

Reply:
It is a name to display in the selector of the execution of the visual Studio
![image](https://user-images.githubusercontent.com/6796523/79398418-0d492080-7f57-11ea-847f-20946e25ca41.png)